internal/registry/registry.go

```diff
@@ -113,8 +112,12 @@ func (r *registry) LastLayer(ctx context.Context, image string, po *kmmv1beta1.P
 func (r *registry) VerifyModuleExists(layer v1.Layer, pathPrefix, kernelVersion, moduleFileName string) bool {
 	// in layers headers, there is no root prefix
 	fullPath := filepath.Join(strings.TrimPrefix(pathPrefix, "/"), modulesLocationPath, kernelVersion, moduleFileName)
-	_, err := r.GetHeaderDataFromLayer(layer, fullPath)
-	return err == nil
+	_, readerCloser, err := r.getHeaderReaderFromLayer(layer, fullPath)
```

Comment:
Can we add a comment saying that if we didn't get any error it means that the file exists? (we are not using the reader so it is less explicit than before).

Reply:
fixed